markdown_lab/core/rust_backend.py

```
@@ -48,18 +48,18 @@ def convert_html_to_format(
         self, html: str, base_url: str, output_format: str = "markdown"
     ) -> str:
         """
-        Convert HTML to the specified format.
-
-        Args:
-            html: HTML content to convert
-            base_url: Base URL for resolving relative links
-            output_format: Target format ("markdown", "json", "xml")
-
+        Converts HTML content to the specified output format using the Rust backend.
+        
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)